### ISSUE TITLE
Create endpoint for slicing zone facts

### ISSUE THREAD

**Issue by NealHumphrey** [codefordc/housing-insights#434]
Create an endpoint that returns matching zones for any desired field from the ZoneFacts table for a user selected zone_type. 

Example: user requests crime_all_12_months for the 'ward' zone type
API response:
```
{
  "data_id": "crime_all_12_months", 
  "grouping": "ward", 
  "items": [
    {
      "count": 4402, 
      "group": "Ward 1"
    }, 
    {
      "count": 5663, 
      "group": "Ward 2"
    }, 
    {
      "count": 1492, 
      "group": "Ward 3"
    }, 
    {
      "count": 2646, 
      "group": "Ward 4"
    }, 
    {
      "count": 4183, 
      "group": "Ward 5"
    }, 
    {
      "count": 5183, 
      "group": "Ward 6"
    }, 
    {
      "count": 3723, 
      "group": "Ward 7"
    }, 
    {
      "count": 3255, 
      "group": "Ward 8"
    }
  ]
}
```

This API endpoint will replace the 'summarize_observations' endpoint in the current API. Creation of various permutations of the calculations (different number of months, different type of crime incidents, etc.) will be done by the calculation of the ZoneFacts table; this endpoint will only need to handle the where zone_type='ward' portion of the query, and formatting of results.

**Comment by NealHumphrey** [comment]
I'm hopping on this now!

**Comment by NealHumphrey** [comment]
Resolved in #440
